cabal-install/tests/IntegrationTests2.hs

```diff
@@ -247,9 +247,15 @@ testTargetSelectors reportSubCase = do
                                  ":pkg:p:lib:p:file:P.y"
                      , "q/QQ.hs", "q:QQ.lhs", "lib:q:QQ.hsc", "q:q:QQ.hsc",
                                   ":pkg:q:lib:q:file:QQ.y"
+                     , "q/Q.hs", "q:Q.lhs", "lib:q:Q.hsc", "q:q:Q.hsc",
+                                  ":pkg:q:lib:q:file:Q.y"
+                     , "app/Main.hs", "p:app/Main.hs", "exe:ppexe:app/Main.hs", "p:ppexe:app/Main.hs",
```

Comment:
Should we also add a test that `Main.hs` would be ambiguous?

Reply:
Yeah, an ambiguous target selector makes sense, going to update it asap!

Reply:
I am sorry, I am not sure I get what you are saying?
Do you mean to add a test for the case that `Main.hs` is ambiguous? Since it should be a filepath, I dont think it can be ambiguous?